<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SOLUTION</class>
 <widget class="QDialog" name="SOLUTION">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelback">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutbutton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>51</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Recce Form:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>741</width>
     <height>401</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="title">
    <string>Verification Needed!</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Base on recce form we found that the enemy location are at</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditlocation_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>29</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>There will be some obstacle which listed beside: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdittask_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdittype_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>type. Which start by</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditstart_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>. In this situation we can use</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdittactic_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>340</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>With the stregth of a</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditstrength_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>There are some weapon been spotted and listed beside:</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>the enemy.With the data retrive we can conclude that is the best to use </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>There is </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditexit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>escape route. The mission is to</string>
    </property>
   </widget>
   <zorder>lineEditlocation_2</zorder>
   <zorder>label_9</zorder>
   <zorder>lineEdittask_2</zorder>
   <zorder>lineEdittype_2</zorder>
   <zorder>label_14</zorder>
   <zorder>lineEditstart_2</zorder>
   <zorder>label_15</zorder>
   <zorder>lineEdittactic_2</zorder>
   <zorder>label_16</zorder>
   <zorder>lineEditstrength_2</zorder>
   <zorder>label_17</zorder>
   <zorder>tableView_3</zorder>
   <zorder>tableView_4</zorder>
   <zorder>label_18</zorder>
   <zorder>label_19</zorder>
   <zorder>label_20</zorder>
   <zorder>lineEditexit_2</zorder>
   <zorder>label_21</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>500</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reason?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
